src/cmd_all/src/bin/risingwave.rs

```diff
@@ -222,6 +223,25 @@ fn main() {
     component.start(&matches);
 }
 
+/// Delegate to the single node mode with the preset `in-memory` option.
+fn playground() {
+    // Ensure that only one playground instance is running at a time.
+    let lock_file = File::create(std::env::temp_dir().join("risingwave-playground-lock")).unwrap();
+    let mut lock = fd_lock::RwLock::new(lock_file);
+    let _guard = loop {
+        match lock.try_write() {
+            Ok(guard) => break guard,
+            Err(err) if matches!(err.kind(), std::io::ErrorKind::WouldBlock) => {
+                eprintln!("ERROR: Another playground instance is running. Please exit it first.");
```

Comment:
Should we inform user where the lock file is? I have encountered scenarios previously where somehow the lockfile is not cleaned.

Then they can clean it on their own.

Reply:
I've tested locally and found that the file will be unlocked no matter how the playground process exited or got killed. 🤔 I guess the scenarios you encountered may have relied on a different mechanism, for example, based on the _existence_ of the lock file instead of occupation.

Reply:
I guess we don't need to explicitly tell user the existence of the file. They should ask on StackOverflow and someone found the file by their own should answer the question😄🤪